# Type: refactor
# Subtype: R10
# English Summary: Delete Logging initialization
# Chinese Summary: 删除 Logging 初始化
# Entry Number: 23

# === Git Diff Content ===

diff --git a/presto-tests/src/main/java/com/facebook/presto/tests/H2QueryRunner.java b/presto-tests/src/main/java/com/facebook/presto/tests/H2QueryRunner.java
index d7d6c43..c992ec3 100644
--- a/presto-tests/src/main/java/com/facebook/presto/tests/H2QueryRunner.java
+++ b/presto-tests/src/main/java/com/facebook/presto/tests/H2QueryRunner.java
@@ -27,7 +27,6 @@ import com.google.common.base.Joiner;
 import com.google.common.base.Predicate;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.Iterables;
-import io.airlift.log.Logging;
 import org.intellij.lang.annotations.Language;
 import org.skife.jdbi.v2.DBI;
 import org.skife.jdbi.v2.Handle;
@@ -69,8 +68,6 @@ public class H2QueryRunner
 
     public H2QueryRunner()
     {
-        Logging.initialize();
-
         handle = DBI.open("jdbc:h2:mem:test" + System.nanoTime());
         TpchMetadata tpchMetadata = new TpchMetadata("");
 
